src/scenes/Home/index.js

```diff
@@ -19,16 +19,16 @@ class Home extends Component {
     }
 
     binance.options({
-      'APIKEY':      props.opts.binance.key,
-      'APISECRET':   props.opts.binance.secret,
-      'recvWindow': 60000
+      APIKEY:      props.opts.binance.key,
+      APISECRET:   props.opts.binance.secret,
+      recvWindow:  60000
     })
   }
 
   componentDidMount() {
     var self = this;
```

Comment:
Remove self reference

Reply:
Home page will be changed in another branche.